modules/gapi/samples/oak_basic_infer.cpp

```diff
@@ -0,0 +1,140 @@
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/infer/parsers.hpp>
+#include <opencv2/gapi/render.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/highgui.hpp>
+
+#include <opencv2/gapi/oak/oak.hpp>
+#include <opencv2/gapi/oak/infer.hpp>
+
+/*
+*                  Disclaimer
+*
+* OAK infer currently has some accuracy issues
+* which are being investigated and fixed.
+* Use the sample and OAK infer at your own risk.
```

Comment:
I'd relax this sentence, at first glance you can think that oak infer can physically break something

Reply:
Done